### ISSUE TITLE
Cannot control hero after reconnecting

### ISSUE THREAD

**Issue by Daar543** [ValveSoftware/Dota2-Gameplay#8898]
### Description

This happens in coop bot matches, not sure if an actual dota game.

If a player disconnects, their hero disappears from the top screen. During that time, a bot can control the hero. But when a player reconnects, the hero reappears on the top screen, but the player is demoted to a spectator. Any items owned by the player are greyed out, as if they were owned by another player. And obviously that makes the unit uncontrollable, cannot even move or attack.

The hero is not assigned to the player either, so builds are bugged too (not recognizing the hero).

**Comment by Daar543** [comment]
Still not fixed.

**Comment by mgheorghe** [comment]
any solution ?

**Comment by Daar543** [comment]
> any solution ?

Not really, if you have 5 party you can set up a lobby (regardless of server) so dc does not transfer the control to bots

**Comment by Brighd** [comment]
Maybe there's a way to escalate this issue? The bug is still there
